pkg/gw/server.go

```diff
@@ -167,8 +167,7 @@ func (s *ImmuGwServer) Start() error {
 	}
 
 	s.installShutdownHandler()
-	s.Logger.Infof("Starting immugw at %s:%d", s.Options.Address, s.Options.Port)
-
+	s.Logger.Infof("starting immudgw: %v", s.Options)
```

Comment:
@mmeloni i think there’s a typo here: shouldn’t it be “immugw” instead of “immuDgw”?

Reply:
Ty